cmd/root.go

```diff
@@ -68,6 +69,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&configFile, "config", "/etc/aurora/australis.yml", "Config file to use.")
 	rootCmd.PersistentFlags().BoolVar(&toJson, "toJSON", false, "Print output in JSON format.")
 	rootCmd.PersistentFlags().StringVarP(&logLevel, "logLevel", "l", "info", "Set logging level ["+internal.GetLoggingLevels()+"].")
+	rootCmd.PersistentFlags().DurationVarP(&timeout, "timeout", "t", 20, "Gorealis timeout (default: 20 seconds)")
```

Comment:
This does not look right.

If someone enters "20s" the value that will be stored in `timeout` will be 20 x 10e9. Then on line 140 that will be multiplied by 10e9 (the value of `time.Second`) and the result will be certainly not be what you expect.

Reply:
good point will fix it.

Reply:
good catch @misho-kr !